EcpBtBot/Conversational_Integration.py

```diff
@@ -1,5 +1,21 @@
-def ReadingMessage():
-    print(u"> ", end=u"")
-    return input()
-def SendingMessage(message):
-    return print(message)
\ No newline at end of file
+from EcpBtBot.MainClass import *
+
+def Receiving_Response(message,userid):
+    #initialisation du sujet
+    if "subject" not in dico_users[userid]:
```

Comment:
i notice that there is a global variable here... `dico_users`. it is usually not a good practice for many reasons (http://wiki.c2.com/?GlobalVariablesAreBad)

So here two things: 
1) you could here have a class, which has a member that have access to the users.
2) what about a database ?? this is a concept that i am not seeing here and that is pretty useful... keep all the data (users, conversations, etc) in a database. I am suggesting a NoSQL database because it's much simpler to use IMO. MongoDB is the main one. Then it means that here, you could have a client, that has a database client, that could connect to it, and see if there is a user with the given id, and decide what to do based on the presence or absence...